perl/holdings_proxy.cgi

```diff
@@ -86,6 +86,51 @@ my $g_callback = '';
       }
     }
   }
+
+  ##################  2017 IX ################  MELINDA-193
+  my $testi = "just testing";
+   my @kids = ();   # Melinda-ID't & 035 ; case MELINDA-193
+     
+   foreach my $field (@$fieldlist)
+    { 
+      # get from field Melinda,  001
+      if ($field->{'code'} eq '001') 
+      {
+           $testi = $field->{'code'}  ;   
+           $testi = $testi . "= " .  $field->{'data'}  ;
+        push(@kids, $field->{'data'});
+      } 
+
+      # get from field 035
+      if ($field->{'code'} eq '035')
+      {
+           $testi = $testi . "\n"  . $field->{'code'}  ;
+           $testi = $testi . "= " .  $field->{'data'}  ;
+        my $found = $field->{'data'}  ;
+           $found =~ s/^....\(FI-MELINDA\)//g ;
+        push(@kids, $found);
+      }
+
+      # get from DATA where there is Melinda number 
+      if ($field->{'data'} =~ /^....\(FI-MELINDA\).*/ ) {             
```

Comment:
035 is a variable data field, so you need to loop the z-subfield to check for the string _FI-MELINDA_